src/main/java/dk/kb/netarchivesuite/solrwayback/parsers/WarcParser.java

```diff
@@ -254,7 +252,7 @@ private static void populateWarcSecondHeader(ArcEntry warcEntry, String headerLi
 
     }  //Content-Length: 31131
     else if (headerLine.toLowerCase().startsWith("content-length:")) {
-      String[] contentLine = headerLine.split(" ");
+      String[] contentLine = headerLine.split(":");
       long totalSize = Long.parseLong(contentLine[1].trim());               
```

Comment:
Under the assumption that the `content-length` follows [RFC 2616](https://www.rfc-editor.org/rfc/rfc2616#page-31), this should work.